resources/assets/js/components/ApplicantProfile/ProfileBasicInformation.tsx

```diff
@@ -0,0 +1,435 @@
+  veteranStatus,
+  gcEmployeeStatus,
+} from "../Application/BasicInfo/basicInfoMessages";
+import { basicInfoMessages } from "../Application/applicationMessages";
+import {
+  CitizenshipId,
+  VeteranId,
+  currentEmployeeIdEnum,
+  currentGcEmployeeId,
+} from "../../models/lookupConstants";
+
+export interface ProfileBasicInformationProps {
+  gocClassifications: GocClassification[];
+  basicInformation: ProfileBasicInformationProp;
+  name: string;
+  email: string;
+}
+
+export interface ClassificationDropdownsProps {
+  gocClassifications: GocClassification[];
+  selectedItem?: GocClassification;
+}
+
+export interface GcExperienceProps {
+  gocClassifications: GocClassification[];
+  previousExperienceProp: GocClassification[];
+  currentGcClassification: GocClassification;
+  wasGcEmployee: number; // 1 == yes, 2 == no, 3 == previous
+}
+
+interface ProfileBasicInformationInitialValues {
+  citizenship: number;
+  veteranStatus: number;
+  currentGcEmployeeStatus: number;
+}
+
+const GcExperience: FunctionComponent<GcExperienceProps> = ({
+  previousExperienceProp,
+  currentGcClassification,
+  gocClassifications,
+  wasGcEmployee,
+}) => {
+  const intl = useIntl();
+
+  const [previousExperience, setPreviousExperience] = useState<
+    GocClassification[]
+  >(previousExperienceProp);
+
+  const removeExperience = function (gocClassification: GocClassification) {
+    setPreviousExperience(
+      previousExperience.filter((experience) => {
+        return !(
+          experience.classification.key ==
+            gocClassification.classification.key &&
+          experience.level == gocClassification.level
+        );
+      }),
+    );
+  };
+
+  const createPreviousExperienceDropdowns = (
+    previousGcExperience: GocClassification[],
+  ) => {
+    return previousGcExperience.map((experience) => {
+      return (
+        <>
+          <li>
+            <div data-c-grid="gutter top">
+              <ClassificationDropdowns
+                gocClassifications={gocClassifications}
+                selectedItem={experience}
+                data-c-grid-item="base(2of2) tl(2of3)"
+              />
+              <div data-c-grid-item="base(1of1) tl(1of3)" data-c-input="select">
+                <button
+                  data-c-button="solid(c1)"
+                  onClick={() => removeExperience(experience)}
+                >
+                  <i className="fa fa-trash" /> Remove
+                </button>
+                <span>
+                  {intl.formatMessage(myBasicInformationMessages.inputError)}
+                </span>
+              </div>
+            </div>
+          </li>
+        </>
+      );
+    });
+  };
+
+  const addExperience = function () {
+    // If the user has not populated their most recent experience, prevent adding a second empty row
+    if (
+      previousExperience.filter((experience) => {
+        return experience.classification.id == 0;
+      }).length > 0
+    ) {
```

Comment:
I'm not sure how, but I think there might be a bug in this logic? When playing with this in Storybook, it sometimes doesn't create a row when I expect it to.
Also, it seems to me it might be nicer UX to disable the button when we don't want it to do anything, rather than abort silently.